src/test/regress/sql/ignoring_orphaned_shards.sql

```diff
@@ -82,9 +82,6 @@ SELECT logicalrelid FROM pg_dist_partition WHERE colocationid = 92448300 ORDER B
 SELECT update_distributed_table_colocation('rep2', 'rep1');
 SELECT logicalrelid FROM pg_dist_partition WHERE colocationid = 92448300 ORDER BY 1;
 
--- cannot copy from an orphaned shard
-SELECT * FROM citus_copy_shard_placement(92448300, 'localhost', :worker_1_port, 'localhost', :master_port);
```

Comment:
This test seems important

Reply:
I deleted that because that orphaned shard gets deleted from that node, so we don't get the orphaned shard error message anymore

Reply:
Makes sense. And we'll make orphaned shards non-existent I guess with a follow up PR, so recreating the tests seems pretty useless.